api-tests/src/steps/ipv-steps.ts

```diff
@@ -382,3 +383,27 @@ When("I call the healthcheck endpoint", async function (this: World) {
 Then("the healthcheck is successful", async function (this: World) {
   assert.ok(this.healthCheckResult);
 });
+
+When("I call the JWKS endpoint", async function (this: World) {
+  this.jwksResult = await externalClient.jwks();
+});
+
+Then("I get a valid JWKS response", async function (this: World) {
```

Comment:
This is a slightly weaker test than the one in ipv-core-tests (which asserts on the exact contents).

IMO this is probably ok - otherwise we'll just have a load of places that need updating if we ever have to do a key rotation.

Reply:
@blakeyp would be good to get a perspective from you here, as I think you were involved in the addition of those tests (after the incident...)

We could still keep the tests in ipv-core-tests, but I'm not sure how useful they are - they don't test production, so if someone was actively tampering then they'd just change prod, not build :shrug:

Reply:
Agree we don't need to assert the exact contents but could we check that the keys array is non-empty? in the incident the endpoint was still up the array was just empty

Reply:
Yeah, that should be tested implicitly by 'must be at least one signing and one encryption key' - but I could add an explicit assertion on length too.

Reply:
Ah cool that looks sufficient